src/plots/plots.js

```diff
@@ -438,17 +438,17 @@ plots.supplyDefaults = function(gd, opts) {
     plots.supplyLayoutModuleDefaults(newLayout, newFullLayout, newFullData, gd._transitionData);
 
     // Special cases that introduce interactions between traces.
-    // This is after relinkPrivateKeys so we can use those in cleanData
+    // This is after relinkPrivateKeys so we can use those in crossTraceDefaults
```

Comment:
Grepping for `cleanData` in streambed, I found:

![image](https://user-images.githubusercontent.com/6675409/46218275-32912200-c312-11e8-9658-ef239558e09d.png)

@VeraZab does this ring a bell? Is that line still getting used?

Reply:
Line 157351 - looks like it's pulling in a whole plotly.js bundle.

Reply:
Good :eye: , false alarm.